util/helm/cmd.go

```diff
@@ -111,17 +111,51 @@ func (c *Cmd) RepoUpdate() (string, error) {
 }
 
 type FetchOpts struct {
-	Version, Destination string
+	Version  string
+	Ca       string
+	Cert     string
+	Password string
+	Username string
 }
 
 func (c *Cmd) Fetch(repo, chartName string, opts FetchOpts) (string, error) {
-	args := []string{"fetch", "--untar", "--untardir", opts.Destination}
+	args := []string{"fetch"}
 
 	if opts.Version != "" {
 		args = append(args, "--version", opts.Version)
 	}
+	if opts.Username != "" {
+		args = append(args, "--username", opts.Username)
+	}
+	if opts.Password != "" {
+		args = append(args, "--password", opts.Password)
+	}
+	if opts.Ca != "" {
```

Comment:
good spot

Reply:
maybe call this after the parameter name for consistency, i.e. `CAFile`

Reply:
How about `CAData`? To make it clear that this is not file name.